HW4_Bredov/protein_analyzer_tool.py

```diff
@@ -0,0 +1,353 @@
+from typing import Iterable, Tuple
+
+AA_TR_DICT = {'Ala': 'A', 'Arg': 'R', 'Asn': 'N', 'Asp': 'D',
+              'Cys': 'C', 'Gln': 'E', 'Glu': 'Q', 'Gly': 'G',
+              'His': 'H', 'Ile': 'I', 'Leu': 'L', 'Lys': 'K',
+              'Met': 'M', 'Phe': 'F', 'Pro': 'P', 'Ser': 'S',
+              'Thr': 'T', 'Trp': 'W', 'Tyr': 'Y', 'Val': 'V'}
+AA_UNIPROT_CONTENT = {
+"A": 9.03, "R": 5.84, "N": 3.79, "D": 5.47, "C": 1.29,
+"Q": 3.80, "E": 6.24, "G": 7.27, "H": 2.22, "I": 5.53,
+"L": 9.85, "K": 4.93, "M": 2.33, "F": 3.88, "P": 4.99,
+"S": 6.82, "T": 5.55, "W": 1.30, "Y": 2.88, "V": 6.86
+}

```

Comment:
Не хватает отступов.
```suggestion
AA_UNIPROT_CONTENT = {"A": 9.03, "R": 5.84, "N": 3.79, "D": 5.47, "C": 1.29,
                      "Q": 3.80, "E": 6.24, "G": 7.27, "H": 2.22, "I": 5.53,
                      "L": 9.85, "K": 4.93, "M": 2.33, "F": 3.88, "P": 4.99,
                      "S": 6.82, "T": 5.55, "W": 1.30, "Y": 2.88, "V": 6.86,
                     }
```